src/test/java/com/google/api/codegen/gapic/GapicTestBase2.java

```diff
@@ -168,6 +175,7 @@ protected boolean suppressDiagnosis() {
       String apiName,
       String protoPackage,
       String clientPackage,
+      String baseline,
```

Comment:
I tried to track down where `baseNames` are used but found that went deep down in `api-compiler` so I gave up. It does not seem to be related to baseline file names we are using to compare diffs.

Reply:
Appears to be a list of stems, and each stem is used to match files that have the stem+suffix, where suffix is stuff like `.proto` and `.yaml`. See https://github.com/googleapis/api-compiler/search?q=baseNames&unscoped_q=baseNames

Might be worth renaming this to `configFilenameStems` in this function.

But please do have a comment clarifying what `baseline` is used for, and how it's autogenerated if not provided. Speaking of which, do we need to autogenerate it or is it cleaner to require it be passed in always?

Reply:
I think it's OK to have keep two overloads of `createConfigs`. An autogenerated name works in most cases (generated from API name and language, e.g., java_library.baseline, python_long_running.baseline). During the migration, we need a separate baseline because we need to be able to generate baselines from two sets of gapic configs, but this won't be needed after the migration is finished.